R/zchunk_LB131.LV_R_GLU.R

```diff
@@ -38,7 +38,7 @@ module_aglu_LB131.LV_R_GLU <- function(command, ...) {
     # Calculate the total value of each geographic land unit (GLU)
     L100.GTAP_LV_milUSD %>%
       left_join_error_no_match(iso_GCAM_regID, by = "iso")  %>%                                         # Map in ISO codes
-      mutate(value = value * CONV_2001_1975_USD) %>%                                                    # Convert to 1975$
+      mutate(value = value * gdp_deflator(1975, base_year = 2001)) %>%                                  # Convert to 1975$
```

Comment:
per slack comment, i think this would be better served with:
mutate(value = value / gdp_deflator(2001, base_year = 1975))